include/vara/Feature/FeatureModelTransaction.h

```diff
@@ -58,7 +63,12 @@ class FeatureModelTransaction
              "Transaction in CopyMode should be commited before destruction.");
     } else {
       if (this->isUncommited()) { // In modification mode we should ensure that
-                                  // changes are commited before destruction
+                                  // changes are committed before destruction
+        llvm::errs()
+            << "warning: Uncommitted modifications before destruction.\n";
+        // TODO(s9latimm): Committing now may break with prior failed commits.
```

Comment:
@s9latimm What do we do here?

Reply:
This strongly depends on se-passau/VaRA/issues/744, as we need a way to track if a prior commit failed to prevent breaking the FM further and prohibit unintended behavior -- this may be combined with a rollback (see se-passau/VaRA/issues/723) while discarding any uncommitted changes.

I kept this open for now and linked the related issue.